src/panels/lovelace/views/hui-view.ts

```diff
@@ -226,59 +226,64 @@ export class HUIView extends LitElement {
       root.removeChild(root.lastChild);
     }
 
-    let columns: [number, number][][] = [];
     const columnEntityCount: number[] = [];
+    const columnElements: HTMLDivElement[] = [];
     for (let i = 0; i < this.columns!; i++) {
```

Comment:
I wonder if we can do a small optimization here that we never make more columns than that we have cards: `Math.min(this.columns, this._cards.length)`.